simpleocr/files.py

```diff
@@ -45,45 +32,152 @@ def write(self):
         write_boxfile(self.path, self.classes, self.segments)
 
 
-class ImageFile(object):
```

Comment:
I think we should probably separate this functionalities. What do you think of having a `Image` class that represents a Image *buffer*, and a `class ImageFile(Image)` that extends it with all the file mechanisms?